test/conftest.py

```diff
@@ -22,6 +22,8 @@ def pytest_addoption(parser):
 def env_setup(monkeypatch):
     monkeypatch.setenv('PATH', f'{os.path.dirname(os.path.abspath(__file__))}/../build',
                        prepend=os.pathsep)
```

Comment:
Why do we keep this?

Reply:
This is a remnant from the version that kept CMake in. Will just use `build/` for meson.